src/components/Info.css

```diff
@@ -0,0 +1,51 @@
+.Info-icon {
+  height: 17px;
+  width: 17px;
+  margin-left: 10px;
+  margin-top: 7px;
+}
+
+.Info-icon:hover {
+  cursor: pointer;
+}
+
+.Info-header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.Info-header h2 {
+  color: #fff;
+}
+
+.Text-container {
+  position: relative;
+}
+
+.Show {
+  position: absolute;
+  z-index: 1;
```

Comment:
`z-index` appears to be unnecessary.

Reply:
Left z-index to make sure text bubble sits on top of graph preview.